.cargo/config.toml

```diff
@@ -22,3 +22,6 @@ xclippy = [
 
 [build]
 rustflags = ["--cfg", "tokio_unstable"]
+
+[target.x86_64-unknown-linux-gnu]
+rustflags = ["--cfg", "tokio_unstable", "-C", "link-arg=-fuse-ld=lld"]
```

Comment:
This may break windows builds.

Reply:
Could you elaborate? I think I'm targeting Linux specifically (x86_64-unknown-linux-gnu).

Reply:
yes, you are good.